Analysis

joannacknight opened the pull request:
Includes all notebooks used to create plots and latex tables for the report.

radka-j commented on scripts/eval_ced.py:
```diff
@@ -87,109 +104,190 @@ def evaluate(
+        .agg(
+            {
+                "MCC": "median",
+                "seed": "first",
```
The first seed does not correspond to the seed of the median model.

joannacknight commented:
I think this is now ready to merge. 

I've removed notebooks where we didn't actually use any of the output in the report. I've also removed the `analysis` subfolder in the `notebooks` folder so all notebooks now in the same folder. I've tidied up a bit of code. The notebooks could do with a few more comments etc. but might address that when doing the ReadMes and instructions.

radka-j reviewed (approved):
LGTM
